001-quickstart-passwordless/Program.cs

```diff
@@ -0,0 +1,78 @@
+﻿// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// ------------------------------------------------------------
+
+// <using_directives> 
+using Microsoft.Azure.Cosmos;
+using Azure.Identity;
+// </using_directives>
+
+// <client_credentials> 
+// New instance of CosmosClient class
+using CosmosClient client = new(
+    "https://<your-cosmosdb-account-name>.documents.azure.com:443/",
+    new DefaultAzureCredential()
```

Comment:
```suggestion
    authKeyOrResourceTokenCredential: new DefaultAzureCredential()
```

I think the @microsoftdocs/dotnet-docs team had us name the parameters to avoid confusion about member overloads.